internal/processor/data.go

```diff
@@ -60,3 +69,44 @@ func processDataSet(dataSet *map[string]interface{}, samplesToMerge *load.Sample
 		CreateMetricSets(mergedData, cfg, i, true, samplesToMerge, originalAPINo)
 	}
 }
+
+func runJq(dataSets interface{}, api load.API) []interface{} {
+	// due to the current design, dataSets is always a slice
+	// so an api that returns a map will still be a slice returned for processing
+	// for better usability if there is no array prefix access the first item in the slice automatically for the user
+	if !strings.HasPrefix(api.Jq, ".[") {
```

Comment:
**Additional scenario:** 
If it is a simple `api.Jq: '.'`, we do not need to add `".[0]"`.
Although it does not do much using `api.Jq: '.'`, it was the first thing i did for the test.

Reply:
resolving as discussed